src/iter/mod.rs

```diff
@@ -30,6 +33,30 @@
 //! iterators work by first constructing a computation and then
 //! executing it.
 //!
+//! In addition to `par_iter()` and friends, some types offer other
+//! ways to create (or consume) parallel iterators:
+//!
+//! - Slices (`&[T]`, `&mut [T]`) offer methods like `par_split` and
+//!   `par_windows`, as well as various parallel sorting
+//!   operations. See [the `ParallelSlice` trait] for the full list.
+//! - Strings (`&str`) offer methods like `par_split` and `par_lines`.
+//!   See [the `ParallelString` trait] for the full list.
+//! - Various collections offer [`par_extend`], which grows a collection
+//!   given a parallel iterator. (You can use [`collect()`] to create a parallel
+//!   iterator.)
+//!
+//! [the `ParallelSlice` trait]: ../slice/trait.ParallelSlice.html
+//! [the `ParallelString` trait]: ../str/trait.ParallelString.html
+//! [`par_extend`]: trait.ParallelExtend.html
+//! [`collect()`]: trait.ParallelIterator.html#method.collect
+//!
+//! These are the full set of methods you can use to create a parallel iterator:
+//!
+//! - `par_iter` -- iterates over `&T` references to the items in the collection
+//! - `par_iter_mut` -- iterates over `&mut T` references to the items in the collection
+//! - `into_par_iter` -- iterates over the items in the collection, taking ownership of each
+//! - `par_split`
```

Comment:
Were you starting to reference `rayon::iter::split()` here?  There is a `par_split()` on slices and strings though.  I'm not sure we can realistically list a "full set of methods" anyway...

Reply:
Hmm yeah I think I had decided to move that complete list into a link to the trait.